osu.Game/Screens/Menu/MenuTip.cs

```diff
@@ -78,49 +80,49 @@ public void ShowNextTip()
             static void formatRegular(SpriteText t) => t.Font = OsuFont.GetFont(size: 16, weight: FontWeight.Regular);
             static void formatSemiBold(SpriteText t) => t.Font = OsuFont.GetFont(size: 16, weight: FontWeight.SemiBold);
 
-            string tip = getRandomTip();
+            var tip = getRandomTip();
 
             textFlow.Clear();
-            textFlow.AddParagraph("a tip for you:", formatSemiBold);
+            textFlow.AddParagraph(MenuTipStrings.MenuTipTitle, formatSemiBold);
             textFlow.AddParagraph(tip, formatRegular);
 
             this.FadeInFromZero(200, Easing.OutQuint)
-                .Delay(1000 + 80 * tip.Length)
+                .Delay(1000 + 80 * tip.ToString().Length)
```

Comment:
You can do better than that, evaluate the string using `LocalisationManager.GetLocalisedString` and read its length.

Reply:
if we're going that far, will also need to halve the length if a character is multi-byte unicode.

Reply:
.NET seems to handle that well for us, on testing with an Arabic character:

![CleanShot 2024-11-25 at 00 22 13](https://github.com/user-attachments/assets/87c9a0e1-eaf0-4ad3-836d-4b53646d9457)

Reply:
Sure.